yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -311,8 +328,19 @@ export class Sequencer extends (EventEmitter as new () => TypedEventEmitter<Sequ
     this.log.verbose(`Created publisher at address ${publisher.getSenderAddress()} for attestor ${attestorAddress}`);
     this.publisher = publisher;
 
-    const coinbase = this.validatorClient!.getCoinbaseForAttestor(attestorAddress);
-    const feeRecipient = this.validatorClient!.getFeeRecipientForAttestor(attestorAddress);
+    // Get proposer credentials
+    // In fisherman mode, use placeholder values since we're building for validation only (empty blocks)
+    let coinbase: EthAddress;
+    let feeRecipient: AztecAddress;
+    if (this.config.fishermanMode) {
+      // Fisherman doesn't have the proposer's keys
+      coinbase = EthAddress.ZERO;
+      feeRecipient = this.config.feeRecipient ?? AztecAddress.ZERO;
+      this.log.debug(`Using placeholder credentials for validation`, { coinbase, feeRecipient });
+    } else {
+      coinbase = this.validatorClient!.getCoinbaseForAttestor(attestorAddress);
+      feeRecipient = this.validatorClient!.getFeeRecipientForAttestor(attestorAddress);
```

Comment:
I believe these two fallback to the attestor address itself if not set. Why not use those, to avoid another if?